src/components/PathwayCourse/C4CApathway/index.js

```diff
@@ -30,9 +30,7 @@ function C4CApathway() {
   const history = useHistory();
 
   useEffect(() => {
-
-if(localStorage.getItem("studentAuth")===null)
-    history.push("/login");
+    if (localStorage.getItem("studentAuth") === null) history.push("/login");
```

Comment:
Tumhen constant refer karnaa chahiye. Aur bhi, tum `!localStorage.getItem("studentAuth")` upayog kar sakte ho kyonki jab chatra logged in, `studentAuth` ko ek truthy value hona chahiye.
```suggestion
    if (!localStorage.getItem("studentAuth")) history.push(PATHS.LOGIN);
```